backend/.gitignore

```diff
@@ -0,0 +1,4 @@
+venv/
+server/static/
+**/__pycache__/
+server/erp_backend/companies.json
```

Comment:
why do you have two gitignores, if you are ony writing backend in this project?

Reply:
server/erp_backend/companies.json is outdated from other project